media/css/sandstone/fira-fonts.less

```diff
@@ -0,0 +1,70 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@font-face {
+    font-family: 'Fira Sans';
+    src: url('/media/fonts/FiraSans-Regular.eot?#iefix') format('embedded-opentype'),
+         url('/media/fonts/FiraSans-Regular.woff2') format('woff2'),
```

Comment:
Recently watched a talk on font performance and usability and in it, the guy suggests putting woff2 first and then woff, otf etc. Perhaps worth considering here?